clang/test/dpct/cmake_migration/case_001/input.cmake

```diff
@@ -20,6 +20,4 @@ project(foo CUDA)
 
 project(foo2 CUDA CXX)
 
-target_link_libraries(foo3 PUBLIC ${CUDA_LIBRARIES} ${TCNN_LIBRARIES} fmt)
```

Comment:
Removing here because target_link_libraries is handled with DPCT helper function

Reply:
Pls send a separate PR to do that.

Reply:
Done.